### DISCUSSION TITLE
Upgrade kubernetes 1.26 -  autoscaling.v2beta2 removed on kubernetes 1.26 and is in the helm chart

### DISCUSSION THREAD

**Discussion by cduv** [apache/airflow#34397] [General]
Having an airflow version  2.2.5 working on kubernetes 1.25  and deployed with helm chart 1.5.0.
I'm looking to update from kubernetes 1.25 to 1.26 and found that autoscaling.v2beta2 is [removed](https://kubernetes.io/docs/reference/using-api/deprecation-guide/#horizontalpodautoscaler-v126) and found that is present in the chart  in https://github.com/apache/airflow/blob/3854264a31ba5c7718c51a034154f12483ad78d3/chart/values.schema.json#L1516 (8 places). As I read in https://helm.sh/docs/topics/charts/#schema-files , this schema validations is run on any helm install and update, and I'm not sure if I could have problems. Does any one already upgraded to 1.26 and could run helm install/upgrade?

**Comment by tsl409** [comment]
I also noticed the reference to the deprecated hpa apiVersion. Will this actually be a problem?
